src/main/java/org/alfresco/repo/web/scripts/BufferedResponse.java

```diff
@@ -261,9 +261,16 @@ else if (outputStream != null)
             {
                 if (logger.isDebugEnabled())
                     logger.debug("Writing Transactional response: size=" + outputStream.getLength());
-                
-                outputStream.flush();
-                FileCopyUtils.copy(outputStream.getInputStream(), res.getOutputStream());
+
+                try
+                {
+                    outputStream.flush();
+                    FileCopyUtils.copy(outputStream.getInputStream(), res.getOutputStream());
+                }
+                finally
```

Comment:
Can the code be refactored to use try-with-resources statement?

Reply:
No. The streams are already closed in FileCOpyUtils.copy.